Localization/de-DE/about.yaml

```
@@ -0,0 +1,12 @@
+---
+url: "https://foldingathome.org/about/"
+language: "de-DE"
+title: "ÜBER"
+text: [
+# Bold text at the start
+"Folding@home (FAH oder F@h)",
+
+# The rest of the paragraph
+#"is a distributed computing project for simulating protein dynamics, including the process of protein folding and the movements of proteins implicated in a variety of diseases. It brings together citizen scientists who volunteer to run simulations of protein dynamics on their personal computers. Insights from this data are helping scientists to better understand biology, and providing new opportunities for developing therapeutics."
+"ist ein Projekt für verteiltes Rechnen zur Simulation von Proteindynamiken, einschließlich des Prozesses der Proteinfaltung und der Bewegung von Proteinen, die innerhalb verschiedenster Erkrankungen eine wichtige Rolle spielen. Es bringt Amateurwissenschaftler zusammen, die sich freiwillig melden um Simulationen dieser Proteindynamik auf ihrem PC laufen zu lassen. Wissenschaftler können durch diese Daten neue Einblicke gewinnen, um Biologie besser zu verstehen und neue Möglichkeiten zur Entwicklung von Therapeutika zu entwickeln."
```

```suggestion
"ist ein Projekt für verteiltes Rechnen zur Simulation von Proteindynamiken, einschließlich des Prozesses der Proteinfaltung und der Bewegung von Proteinen, die innerhalb verschiedenster Erkrankungen eine wichtige Rolle spielen. Es bringt Amateurwissenschaftler zusammen, die freiwillig Simulationen dieser Proteindynamik auf ihrem PC laufen lassen. Wissenschaftler können durch diese Daten neue Einblicke gewinnen, um Biologie besser zu verstehen und neue Möglichkeiten zur Entwicklung von Therapeutika zu entwickeln."
```
"Freiwillig melden" sounds little too involved for me. Starting to contribute is super easy and requires no interaction with anyone. The current translation sounds a bit like that to me.

---

I also agree to that

---

is going to be in the next commit.